automation/build-artifacts.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
```

Comment:
Please integrate with build-rpm.sh provided in https://github.com/oVirt/ovirt-engine-keycloak/pull/4